pkg/adapter/mtping/runner_test.go

```diff
@@ -198,7 +198,7 @@ func validateSent(t *testing.T, ce *adaptertesting.TestCloudEventsClient, wantDa
 	gotExtensions := ce.Sent()[0].Context.GetExtensions()
 
 	if extensions == nil && gotExtensions != nil {
-		t.Errorf("Expected event with no extension overrides got %v", gotExtensions)
+		t.Error("Expected event with no extension overrides got", gotExtensions)
```

Comment:
nit: use similar punctuation for test logs
```suggestion
		t.Error("Expected event with no extension overrides, got", gotExtensions)
```

Reply:
This is a bit hard in a script but fixed.